<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>490</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_postView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>331</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>내용</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_title">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>제목</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_dept">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>작성부서</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_writer">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>작성자</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_date">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>날짜</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_views">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>275</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>조회수</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_checkBoard">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>32</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>지금 게시판 체크</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>현재 게시글 번호: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_currPostNum">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x좌표: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mouseXpos">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>80</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>143</x>
     <y>80</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>y좌표: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_mouseYpos">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>105</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>카카오톡 채팅창 텍스트 영역:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sendToKakao">
   <property name="geometry">
    <rect>
     <x>193</x>
     <y>140</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>카카오톡 발송</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_kakaoXpos">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
   <property name="value">
    <number>1300</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_kakaoYpos">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
   <property name="value">
    <number>610</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>145</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>디버그:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>65</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_loadPost">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>216</y>
     <width>94</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>불러오기</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>220</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>게시글 URL</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_URL">
   <property name="geometry">
    <rect>
     <x>87</x>
     <y>220</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>https://www.ajou.ac.kr/kr/ajou/notice.do?mode=view&amp;articleNo=112225&amp;article.offset=0&amp;articleLimit=10</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>새로고침 주기(초): </string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_updateCycle">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>35</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <number>5</number>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
   <property name="value">
    <number>60</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>새로고침 남은 시간(초): </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_updatedTime">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pauseTimer">
   <property name="geometry">
    <rect>
     <x>193</x>
     <y>170</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>타이머 일시정지</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_openImagePath">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>이미지 저장 경로 열기</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>spinBox_updateCycle</tabstop>
  <tabstop>pushButton_checkBoard</tabstop>
  <tabstop>spinBox_kakaoXpos</tabstop>
  <tabstop>spinBox_kakaoYpos</tabstop>
  <tabstop>pushButton_sendToKakao</tabstop>
  <tabstop>lineEdit_URL</tabstop>
  <tabstop>pushButton_loadPost</tabstop>
  <tabstop>checkBox_text</tabstop>
  <tabstop>checkBox_title</tabstop>
  <tabstop>checkBox_dept</tabstop>
  <tabstop>checkBox_writer</tabstop>
  <tabstop>checkBox_date</tabstop>
  <tabstop>checkBox_views</tabstop>
  <tabstop>textBrowser_postView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
